### DISCUSSION TITLE
Training process feedback

### DISCUSSION THREAD

**Discussion by dschrempf** [alan-turing-institute/sktime#7421] [General]
Hello!

Thanks for this amazing package!

We are using `sktime` to classify time series data. We have between 1k and 13k instances with 1 up to around 60 channels of around 1k length. That is, our data roughly has a maximum shape of (13k,60,1k), but many times it is actually smaller. In general, we observe very long training times, even for supposedly fast classifiers such as MiniRocket.

*Feedback while training the classifier* would be of great help to us. That is: How is the training going? What is the ETA? Many times, we start the training process for a classifier, wait for an hour and cancel the run because we have no idea if we have to wait another 5 minutes or 2 days.

Some classifiers have `verbose=1` settings, but those are not helpful. Then we see a ton of `numba` debug output, but at some point the output stops for an unknown amount of time (can be hours).

Maybe we are not aware of existing features. If so, how do you get feedback while training classifiers?

I am looking forward to your replies!

**Comment by fkiraly** [comment]
Thanks for the suggestion! I am moving this to a feature request issue so it can be worked on!

**Reply by dschrempf** [reply]
Thank you!
